Makefile

```diff
@@ -36,9 +36,9 @@ test-unit:
 test-e2e:
 	./test/e2e-tests.sh
 
-# TODO: that will - soon... run the Kafka operator e2e hook
+# Run the Kafka operator e2e hook
 test-e2e-kafka:
```

Comment:
can you rename to `test-e2e-with-kafka` ? :grin:

Reply:
done in 812866d